shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -2082,17 +2083,17 @@ - (BOOL)shouldAutorotate {
 }
 
 - (NSUInteger)supportedInterfaceOrientations {
-  return _orientationPreferences;
+  return self.orientationPreferences;
 }
 
 #pragma mark - Accessibility
 
 - (void)onAccessibilityStatusChanged:(NSNotification*)notification {
-  if (!_engine) {
+  if (!self.engine) {
     return;
   }
-  auto platformView = [_engine.get() platformView];
-  int32_t flags = [self accessibilityFlags];
+  fml::WeakPtr<flutter::PlatformView> platformView = self.engine.platformView;
```

Comment:
Not part of ARC migration and unnecessary in this patch, but making the type obvious here is nice just so people understand upfront that it's one of our smart pointer types.